### PR TITLE
create the GUI for game over screen

### PR DESCRIPTION
**Pull request by MidnattLantern**

### CODE REVIEW

File: assets/css/play-page.css [hunk 1]
```diff
@@ -225,3 +225,62 @@ object{ /* vector graphics can distrupt input, keep this to none */
         height: 150px;
     }
 }
+
+/* --- game over --- */
+.gameOverContainer {
```
Comment (Dee-McG) [inline assets/css/play-page.css]: Can we stick to kebab case for class and ids as per standard? These are camel case which is fine for javascript vars and functions

Comment (MidnattLantern) [reply assets/css/play-page.css]: I'll change that

Comment (MidnattLantern) [reply assets/css/play-page.css]: I have changed from camelCase to kebab

Comment (MidnattLantern) [reply assets/css/play-page.css]: I also added a favicon

**Review by Dee-McG — APPROVED** [review]

**Review by Dee-McG — APPROVED** [review]

**Review by KC-85 — APPROVED** [review]

**Pull request merged**